config/projects.yml

```diff
@@ -80,6 +80,13 @@ classification-bundle:
         symfony: ['2.3', '2.7', '2.8', '3.0', '3.1']
         sonata_admin: ['3']
 
+classification-media-bundle:
+  branches:
+    master:
+      php: ['5.6', '7.0']
```

Comment:
No, because the new bundle would only support the latest master/4.0 version of the dependencies.

Supporting the 3.x branches too would be a lot of pain...